.github/workflows/integration-windows.yml

```diff
@@ -28,6 +28,6 @@ jobs:
         fetch-depth: 0
 
     # TODO(aaron-prindle) skip integration tests for doc only changes on Windows.  Figure out the proper syntax, etc.
-    - name: Run windows unit tests and integration tests
+    - name: Run skaffold unit tests on windows
```

Comment:
do you want to use the same packages as darwin integration test?
```
  go test -short -timeout 60s  ./pkg/skaffold/... ./cmd/... ./hack/... ./pkg/webhook/...
```

Reply:
I can update this to be the same as `darwin`.  For more context here, this is the same as what was in the travis.yml setup for windows as is currently:
```
    - os: windows
      name: "Windows unit"
      script:
        - go test -short -timeout 60s ./...
```

Reply:
I've updated the windows ci tests to do a smaller subset of unit tests (the same as `darwin` as suggested)

Reply:
ah. I wonder how things were working before. 🤷